config.ini

```diff
@@ -69,15 +77,17 @@ mono_font = Inconsolata
 
 [library]
 
-# Where code blocks will save to
-save_filename = d2l/d2l.py
+# A list of filename and pattern pairs.
+save_patterns = d2l/mxnet.py, mxnet
+                d2l/torch.py, pytorch
+                d2l/tensorflow.py, tensorflow
+
+version = 0.13.2
 
-# The parttern to mark this block will be saved.
-save_mark = Saved in the d2l package for later use
+version_file = d2l/__init__.py
 
 
 [deploy]
-s3_bucket = s3://en.d2l.ai
```

Comment:
Do we want to keep it when merging?

Reply:
it's not needed anymore, and it's not safe if others copy it